ui/components/SourceDetail.tsx

```diff
@@ -23,6 +23,24 @@ type Props = {
   info: <T>(s: T) => InfoField[];
 };
 
+type StatusProps = {
+  status?: string;
+  msg: string;
+};
+const PageStatus = ({ status, msg }: StatusProps) => {
```

Comment:
This needs to be repeated on automations as well right?

Reply:
Yeah it needs to be done on everything that doesn't have source table - I made a new issue for it but I should just separate this out here.